src/OrbitGl/TimeGraph.cpp

```diff
@@ -630,21 +695,112 @@ void TimeGraph::DoUpdateLayout() {
 }
 
 void TimeGraph::UpdateChildrenPosAndContainerSize() {
-  // Special case: TimeGraph will set TrackContainer height based on its free space.
-  float total_height_without_track_container = 0;
-  for (orbit_gl::CaptureViewElement* child : GetNonHiddenChildren()) {
-    if (child != track_container_.get()) {
-      total_height_without_track_container += child->GetHeight();
-    }
-  }
-  track_container_->SetHeight(GetHeight() - total_height_without_track_container);
-
-  // Update position of visible children.
-  float current_pos_y = GetPos()[1];
-  for (orbit_gl::CaptureViewElement* child : GetNonHiddenChildren()) {
-    child->SetPos(GetPos()[0], current_pos_y);
-    current_pos_y += child->GetHeight();
-  }
+  // TimeGraph's children (if vertical slider is visible)
```

Comment:
This is really nice.

Reply:
What's the reason for the margin between track container and the slider?

Reply:
Just out of curiosity, it feels inconsistent in the diagram (since there is a margin to the right, but not at the bottom). No need to change anything though.

Reply:
Yes, it's a bit incosistent. I will ask Carsten. 
- There is no margin at the left, but because there is no elements at the left. 
- At the top of the Scheduler we have a margin to not mix with the Timeline (some Tracks are using similar background color than the timeline)
- And at the bottom we have a margin after the last track, which I liked it but maybe because I'm used to.

Reply:
After a talk with Carsten, he agreed with the current margin's way as a temporal state and suggested the top one to be a bit smaller.